lib/rules/lighter-video-files.js

```diff
@@ -0,0 +1,71 @@
+/**
+* @fileoverview Rule to avoid heavy video file formats
+* @author Martyna Babiak
+*/
+
+"use strict";
+
+//------------------------------------------------------------------------------
+// Requirements
+//------------------------------------------------------------------------------
+
+const moduleVisitor = require("eslint-module-utils/moduleVisitor");
+
+//------------------------------------------------------------------------------
+// Rule Definition
+//------------------------------------------------------------------------------
+
+/** @type {import('../shared/types').Rule} */
+module.exports = {
+    meta: {
+        type: "problem",
+
+        docs: {
+            description: "Avoid heavy formats of video files",
+            category: "Code improvements",
+            recommended: false,
+            url: "https://eslint.org/docs/rules/lighter-video-files"
+        },
+        messages: {
+          rejected: 'Format of video file can be changed to WebM. CO2 reduction: even 60% of the video file.\n' +
```

Comment:
pls remove CO2 reduction part - there's 'estimated CO2 reduction' later on, so there's no need to duplicate (it was changed recently for other rules too)

Reply:
you can check lighter-image-files for reference